internal/domain/models/feedback.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// ChatGPT's tip for creating separate model for input.
+type FeedbackInput struct {
+	CustomerName string `json:"customer_name"` //nolint:tagliatelle
+	Email        string `json:"email"`
+	FeedbackText string `json:"feedback_text"` //nolint:tagliatelle
+	Source       string `json:"source"`
+}
+
+type Feedback struct {
+	ID           uuid.UUID    `json:"id" gorm:"type:uuid;primaryKey"`
+	CustomerName string    `json:"customer_name"` //nolint:tagliatelle
```

Comment:
We can remove `nolint` here and configure `tagliatelle` for [the snake case](https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml#L1720) by adding these lines to `.golangci.yml`:

```
linters-settings:
  tagliatelle:
    case:
      rules:
        - json: snake
```